apps/frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,74 @@
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+	type InputOption,
+} from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T, null>;
+	label: string;
+	name: FieldPath<T>;
+	options?: InputOption[];
```

Comment:
I believe options should be a mandatory field, otherwise what's the purpose of this component?